From: Phil Elwell <phil@raspberrypi.com>
Date: Tue, 25 Aug 2020 14:19:51 +0100
Subject: Revert "ARM: dts: bcm2711: Add mmc_ddr_1_8v where possible"

The commit being reverted has been seen to prevent booting with an eMMC
supposedly capable of running at 1.8V in an SD adaptor.

Signed-off-by: Phil Elwell <phil@raspberrypi.com>
---
 arch/arm/boot/dts/bcm2711-rpi-4-b.dts | 1 -
 arch/arm/boot/dts/bcm2711-rpi-cm4.dts | 3 +--
 2 files changed, 1 insertion(+), 3 deletions(-)

diff --git a/arch/arm/boot/dts/bcm2711-rpi-4-b.dts b/arch/arm/boot/dts/bcm2711-rpi-4-b.dts
index 5469f5ad4bb3..21b20e334b1a 100644
--- a/arch/arm/boot/dts/bcm2711-rpi-4-b.dts
+++ b/arch/arm/boot/dts/bcm2711-rpi-4-b.dts
@@ -173,7 +173,6 @@
 /* EMMC2 is used to drive the SD card */
 &emmc2 {
 	vqmmc-supply = <&sd_io_1v8_reg>;
-	mmc-ddr-1_8v;
 	broken-cd;
 	status = "okay";
 };
diff --git a/arch/arm/boot/dts/bcm2711-rpi-cm4.dts b/arch/arm/boot/dts/bcm2711-rpi-cm4.dts
index 823aeeca393d..d4ea928e30ce 100644
--- a/arch/arm/boot/dts/bcm2711-rpi-cm4.dts
+++ b/arch/arm/boot/dts/bcm2711-rpi-cm4.dts
@@ -109,10 +109,9 @@
 	};
 };
 
-/* EMMC2 is used to drive the eMMC/SD card */
+/* EMMC2 is used to drive the SD card */
 &emmc2 {
 	vqmmc-supply = <&sd_io_1v8_reg>;
-	mmc-ddr-1_8v;
 	broken-cd;
 	status = "okay";
 };
